docs/reference/api/security/TLS.md

```diff
@@ -45,20 +45,20 @@ Then create a file named `mbed_app.json` at the root of your application with th
 
 Some hardware accelerators require initialization, regardless of the specific cryptography engine. For this, we introduced `mbedtls_platform_setup()` and `mbedtls_platform_terminate()`.
 
-As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. We suggest you set the platform context parameter for these functions as `NULL`.
+As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. For readability reasons, we suggest you set the platform context parameter for these functions as `NULL`, as it is not being used in Mbed OS.
 
 ```
-    int exit_code = MBEDTLS_EXIT_FAILURE;
+    int ret = 0;
 
-    if((exit_code = mbedtls_platform_setup(NULL)) != 0) {
-        mbedtls_printf("Platform initialization failed with error %d\r\n", exit_code);
-        return MBEDTLS_EXIT_FAILURE;
+    if((ret = mbedtls_platform_setup(NULL)) != 0) {
+        mbedtls_printf("Platform initialization failed with error %d\r\n", ret);
+        return ret;
```

Comment:
Returning `1` here would be safer, as, in general, we should not return Mbed TLS error codes from main (as they may exceed the valid range handled by an OS). I have not heard of it ever being an actual issue, apart from conforming to standards (such as POSIX) or not. You may consider changing it, but I don't find it necessary.